lib/backpex/live_components/form_component.ex

```diff
@@ -328,7 +328,7 @@ defmodule Backpex.FormComponent do
       %{valid?: true} ->
         result = resource_action.module.handle(socket, params)
 
-        if match?({:ok, _msg}, result), do: handle_uploads(socket)
+        if match?({:ok, _msg}, result), do: handle_uploads(socket, params)
```

Comment:
We do not have an item here so what do we want to pass to `handle_uploads`?

Reply:
We now use `Ecto.Changeset.apply_action/2` and pass the data instead of the params